client/app/scripts/superdesk-packaging/views/sd-package-item-preview.html

```diff
@@ -17,7 +17,7 @@
             <div ng-hide=":: item.headline || item.slugline || item.description" translate>Blank headline received</div>
         </div>
 
-        <a ng-href="#/{{ :: type !== 'composite' ? 'authoring' : 'packaging'}}/{{ :: item.guid }}/view?_id={{ :: item.guid }}" 
+        <a ng-href="#/{{ :: type !== 'composite' ? 'authoring' : 'packaging'}}/{{ :: type !== 'video' || 'picture' ? item.residRef : item.guid}}/view?_id={{ :: item.guid }}" 
```

Comment:
Sorry, you were right :D ... When I was writing I was accidentally added !== instead of === ... But I found that for text elements item.residRef is equal to item.quid ... it is different just for picture and video items... So I synchronized code now without condition.